test/integration/script-loader/test/index.test.js

```diff
@@ -12,21 +12,27 @@ import {
 import webdriver from 'next-webdriver'
```

Comment:
Can the `public/.gitkeep` be removed above now that the public folder isn't used?

Reply:
Ah yes, forgot to remove it 😅 

Removed!